crates/aggchain-proof-service/src/custom_chain_data/mod.rs

```diff
@@ -0,0 +1,41 @@
+use aggchain_proof_core::{Digest, AGGCHAIN_PROOF_PROGRAM_VERSION, AGGCHAIN_TYPE};
+use alloy_primitives::U256;
+use bincode::Options;
+use serde::{Deserialize, Serialize};
+
+#[cfg(test)]
+mod tests;
+
+type VKeySelector = [u8; 4];
```

Comment:
Nit: I'd prefer this to be a strong type rather than alias. Would allow you to replace the freestanding function `compute_aggchain_vkey_selector` with something like `VKeySelector::new`.